src/backend/marsha/core/utils/cloudfront_utils.py

```diff
@@ -39,4 +39,6 @@ def rsa_signer(message):
     except FileNotFoundError:
         raise MissingRSAKey()
 
-    return private_key.sign(message, padding.PKCS1v15(), hashes.SHA1())
+    # The following line is excluded from bandit security check because cloudfront supports
+    # only sha1 hash for signed URLs.
```

Comment:
actually the new v4 signature support sha256 : https://docs.aws.amazon.com/general/latest/gr/sigv4-signed-request-examples.html which may or may not be too much for this PR :man_shrugging:

Reply:
Oh, good catch ! I didn't find this one..

I would be reassured if the switch to v4 signatures with sha256 was done in another PR, to ensure that there is no regression.

Reply:
According to this [documentation](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-overview.html) there is no `sha256` signature for cloudfront:
> You must use RSA-SHA1 for signing URLs or cookies. CloudFront doesn't accept other algorithms.

I think this PR can be merged without modification